.devcontainer/Dockerfile

```diff
@@ -17,13 +17,4 @@ RUN yes | unminimize 2>&1 \
 
 # # In this instance we add the psql command so we can access the database from the command line
 RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get -y install --no-install-recommends postgresql-client 
-
-# [Choice] Ubuntu version (use ubuntu-22.04 or ubuntu-18.04 on local arm64/Apple Silicon): ubuntu-22.04, ubuntu-20.04, ubuntu-18.04
-# ARG VARIANT=ubuntu-20.04
-# FROM mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
-
-# [Optional] Uncomment this section to install additional OS packages.
-# In this instance we add the psql command so we can access the database from the command line
-# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-#     && apt-get -y install --no-install-recommends postgresql-client 
\ No newline at end of file
+    && apt-get -y install --no-install-recommends postgresql-client leiningen
```

Comment:
We don't use leiningen any more, so that can be removed.

Quick explanation: Lein was the standard (community driven) build tool since early Clojure days, but has since been largely superseded by the official Clojure CLI by the folks at Cognitect (creators/maintainers of Clojure). We've moved over to this system, as configured through `deps.edn`, and the old `project.clj` can/should be removed now.

Reply:
Great thanks that's really useful to know - removed